### PR TITLE
Buy Now QA Fixes

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR is for [PURCHASE-561](https://artsyproduct.atlassian.net/browse/PURCHASE-561) and addresses the following QA items:

- Corrects case of "RWANDA"
- Several copy changes
- Changing email address for Order app errors to orders@artsy.net (in a way that makes ErrorModal still usable from non-Orders components)
- Makes the error modal narrow (changes `isWide` prop of `ModalContainer` to an enum called `width`, let me know what you think)
- Adds some stories to make testing these changes easier

I've kept these in individual commits so that we can cherry-pick some if one change is blocking or something.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit a5071bc4003c6924fd92933ff001421aed94e38b

https://deploy-preview-1410--artsy-reaction.netlify.com

**Comment by ashfurrow** [comment]
Looks like an issue with some ternary logic – will fix.

File: src/Components/Modal/ModalWrapper.tsx [hunk 1]
```diff
@@ -5,11 +5,17 @@ import FadeTransition from "../Animation/FadeTransition"
 import { media } from "../Helpers"
 import { CtaProps } from "./ModalCta"
 
+export enum ModalWidth {
+  Narrow = "280px",
+  Normal = "900px",
+  Wide = "440px",
```
Comment (ashfurrow) [inline src/Components/Modal/ModalWrapper.tsx]: Normal and Wide are mixed up here, leading to the problem @sweir27 described 😂

**Comment by ashfurrow** [comment]
@sweir27 this looks good to me now, let me know if we're good to merge 👍

**Comment by sweir27** [comment]
Bahaha good catch! 🎉

**Pull request merged**